fractions/wildfly/security/module.conf

```diff
@@ -1,2 +1,4 @@
 org.jboss.as.security
 org.jboss.security.negotiation
+org.wildfly.swarm.ejb
```

Comment:
i think both of these need to be optional otherwise they will always be added to uber jar when `security` is included